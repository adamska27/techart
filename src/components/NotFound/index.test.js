import React from 'react';
import renderer from 'react-test-renderer';

import Component from './index';

describe('NotFound', () => {
  it('NotFound should match the snapshot', () => {
    const component = renderer.create(<Component />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
