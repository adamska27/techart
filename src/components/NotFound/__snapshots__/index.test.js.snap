// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound NotFound should match the snapshot 1`] = `
<div>
  <h2>
    La page n'existe pas ! :(
  </h2>
</div>
`;
